### ISSUE TITLE
Project vcpkg settings missing from Visual Studio Project Properties

### ISSUE THREAD

**Issue by jleaders** [microsoft/vcpkg#18179]
One of my C++ vcxproj's (with a vcpkg.json) looks like this 
![1ONIl](https://user-images.githubusercontent.com/2430917/120054174-44f01c00-bfe3-11eb-80e9-9732e351d986.png)

And another dependant project (with a vcpkg.json) looks like this. Note the missing option.

![5dLef](https://user-images.githubusercontent.com/2430917/120054178-46b9df80-bfe3-11eb-9f8e-b19fcfa849b3.png)

But I need to be able to enable the vcpkg Manifest so it can compile! I'm stuck, where did this option go? Both proj's are in the same solution. Both have vcpkg.json added to their Source Files. Notably one has a different dir structure (proj/proj.vcxproj vs src/lib/proj/build/proj.vcxproj) so the vcpkg.json is further away

As a design note, conditionally hidden options or grayed-out options are an anti-pattern.

**Comment by JackBoosY** [comment]
For msvc project, vcpkg will add extra option `vcpkg` to all project configuration pages.
But for cmake project, since the `sln` is generated by cmake and we always modify the configuration in CMakeLists.txt, we can only use vcpkg toolchain in vcpkg and skip add the option to the configuration pages.

**Comment by jleaders** [comment]
Okay, I found out how to fix it. I had gotten myself in a corrupted state by unloading and loading VS projects while VS was open AND while running `vcpkg integrate project/install` commands.

 I misunderstood those commands. I thought `vcpkg integrate project` installed MSBuild vcpkg just for that project so I thought I had to run that for each .vcprojx. 
Rather, `vcpkg integrate project` may be better named `vcpkg integrate nuget` and `vcpkg integrate install` could be named `vcpkg integrate msbuild`?

In any case, the fix was to close all instances of Visual Studio (I even uninstalled extra versions), and just run `vcpkg integrate install` ONCE and then reopen Visual Studio. Then it worked.
